src/index.html

```diff
@@ -5,6 +5,7 @@
     ${require('./partials/header.html')}
     <main class="content">
         ${require('./partials/presentation.html')}
+        ${require('./partials/news.html')}
```

Comment:
we should put the news down in the page and have a special 'news' page in the menu